CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.33.0 - 20223-09-21
+
+- Feature: Adds MAESTRO_DRIVER_STARTUP_TIMEOUT to iOS driver to configure timeout to start iOS driver, used in CI/CD environment with performance limitations. Thanks, @Jesse Farsong for contributing.
```

Comment:
Is this the right GitHub handle? If not, let's just use the name

Reply:
Nope its name, @ was typo there